source/minimessage/dynamic-replacements.rst

```diff
@@ -46,6 +46,32 @@ The parsed placeholder will insert the replacement before parsing the string. Th
 
 This will insert and parse the text.
 
+Insert a style
+--------------
+
+When you want to create your own styling tag you can use the styling placeholder.
+
+.. code:: java
+
+  MiniMessage.miniMessage().deserialize("<my-style>Hello :)</my-style> How are you?",
+      Placeholder.styling("my-style", ClickEvent.suggestCommand("/say hello"), NamedTextColor.RED, TextDecoration.BOLD));
+  // will apply a click even, a red text color and bold decoration to the text
+
+This will insert the style ith a click event and a red text. Styling placeholders can be used for any style, e.g. colors, text decoration and events.
```

Comment:
```suggestion
This will insert the style with a click event and a red text. Styling placeholders can be used for any style, e.g. colors, text decoration and events.
```